mlfinlab/feature_importance/fingerpint.py

```diff
@@ -0,0 +1,181 @@
+"""
+Implements model fingerprint algorithm from 'Beyond the Black Box' paper
+https://jfds.pm-research.com/content/early/2019/12/11/jfds.2019.1.023
+"""
+
+import pandas as pd
+import numpy as np
+from sklearn.linear_model import LinearRegression
+import matplotlib.pyplot as plt
+
+
+# pylint: disable=invalid-name
+
+class RegressionModelFingerprint:
+    """
+    Regression Fingerprint class
```

Comment:
Lets expand on the docstring here to talk about the paper, that it decomposes into 3 pieces, add link to paper.